lib/routers/profile/person.js

```diff
@@ -100,6 +100,18 @@ router.get('/', (req, res, next) => {
   Promise.resolve()
     .then(() => getSingleProfile(req))
     .then(profile => {
+      if (profile && profile.certificates) {
+        profile.certificates = profile.certificates.map(cert => {
+          if (cert.species && cert.species.length) {
+            return cert;
+          }
+          return {
+            ...cert,
+            species: (cert.modules || [])
+              .reduce((arr, module) => [ ...arr, ...(module.species || []) ], [])
```

Comment:
can we extract this reduce function and give it a meaningful name please?

Reply:
for you, anything x